app/assets/stylesheets/plugins/wice_grid.scss

```diff
@@ -1,127 +1,150 @@
       margin-left: 5px;
       vertical-align: top;
+      width: 10px;
     }
 
-    .collapse-multi-select-icon{
+    .collapse-multi-select-icon {
       background: transparent asset-url("/assets/icons/grid/collapse.gif") no-repeat;
     }
 
-    .expand-multi-select-icon{
+    .expand-multi-select-icon {
       background: transparent asset-url("/assets/icons/grid/expand.gif") no-repeat;
     }
   }
 }
 
-.wice-grid{
-  .desc,  .asc{
+.wice-grid {
+  .desc,
+  .asc {
     padding-right: 18px;
-    text-decoration:none;
+    text-decoration: none;
   }
 
   .desc {
     background: transparent asset-url("/assets/icons/grid/arrow_down.gif") right no-repeati;
   }
-  .asc  {
+
+  .asc {
     background: transparent asset-url("/assets/icons/grid/arrow_up.gif") right no-repeat;
   }
 
-  .submit.clickable{
+  .submit.clickable {
     @include icon-dimensions;
     background: transparent asset-url("/assets/icons/grid/table_refresh.png") no-repeat;
   }
 
-  .reset.clickable{
+  .reset.clickable {
     @include icon-dimensions;
     background: transparent asset-url("/assets/icons/grid/table.png") no-repeat;
   }
 
   .wg-show-filter.clickable,
-  .wg-hide-filter.clickable{
+  .wg-hide-filter.clickable {
     @include icon-dimensions;
     background: transparent asset-url("/assets/icons/grid/page_white_find.png") no-repeat;
   }
 
-  .export-to-csv-button.clickable{
+  .export-to-csv-button.clickable {
     @include icon-dimensions;
     background: transparent asset-url("/assets/icons/grid/page_white_excel.png") no-repeat;
   }
 
-  .clickable.select-all{
+  .clickable.select-all {
     @include icon-dimensions;
     background: transparent asset-url("/assets/icons/grid/tick_all.png") no-repeat;
-    float:left;
+    float: left;
   }
 
-  .clickable.deselect-all{
+  .clickable.deselect-all {
     @include icon-dimensions;
     background: transparent asset-url("/assets/icons/grid/untick_all.png") no-repeat;
-    float:left;
+    float: left;
   }
 
-  /* in case of twitter bootstrap :) */
-  thead th select{
+  // in case of twitter bootstrap :)
+  thead th select {
     width: auto;
   }
 
-  .pagination{
-    margin: 0px;
+  .pagination {
     float: left;
+    margin: 0;
   }
 
-  tr.wg-filter-row  input[type=text] {
-    width: 100px;
+  tr {
+    .wg-filter-row input[type=text] {
+      width: 100px;
+    }
   }
 
   .pagination_status {
-    font-weight: bold;
     float: right;
+    font-weight: bold;
   }
 }
 
-.wice-grid-query-panel{
-  li {list-style-type: none; }
+.wice-grid-query-panel {
+  li {
+    list-style-type: none;
+  }
+
+  ul {
+    margin-left: 0;
+  }
 
-  ul {margin-left: 0 }
+  a {
+    .wice-grid-delete-query .delete-icon {
 
-  a.wice-grid-delete-query .delete-icon{
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/delete.png") no-repeat;
-    float: left;
+      @include icon-dimensions;
+      background: transparent asset-url("/assets/icons/grid/delete.png") no-repeat;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.